File: back-end/src/main/java/com/spring/resturant/model/Item.java
package com.spring.resturant.model;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "item")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Item extends BaseEntity{

    @Column(name = "image")
    private String img;

    @Column(name = "price")
    private int price;

    @Column(name = "quantity")
    private int quantity;

    @ManyToOne
    @JoinColumn(name="request_order_id")
    private RequestOrder requestOrder;

}

File: front-end/src/app/model/item.ts
import {CartOrder} from './cart-order';

export class Item {

  img: string;
  quantity: number;
  price: number;

  constructor(cartOrder: CartOrder) {
    this.img  = cartOrder.img;
    this.quantity = cartOrder.quantity;
    this.price = cartOrder.price
  }
}
